### ISSUE TITLE
r.minval/maxval does not work with a compound index

### ISSUE THREAD

**Issue by simonratner** [neumino/reqlite#73]
Given a compound index, such as `[r.row('foo'), r.row('bar')]`, this query never returns:
```js
r.db(...).table(...).between([1, r.minval], [1, r.maxval], {index: 'foobar'})
```

**Comment by simonratner** [comment]
I haven't tried other permutations, such as individual bounds.

**Comment by neumino** [comment]
Hum, this is really weird, looking into it.

**Comment by neumino** [comment]
Thanks for the test case too :)

**Comment by neumino** [comment]
That should be fixed in 2.2.8. `r.minval` was not returning a promise in reqlite and the process was crashing when calling `.then` on it.
I also found another sneaky bug while looking at that about ordering sequences with datum we can't return (like `r.minval`)
